packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -167,6 +172,7 @@ abstract contract CrossDomainMessenger is
         );
 
         emit SentMessage(_target, msg.sender, _message, messageNonce(), _minGasLimit);
+        emit SentMessageExtraData(msg.sender, msg.value);
```

Comment:
SentMessage is a legacy event that shouldn't be changed so we can preserve the interface. Instead of adding a completely new event, we add `SentMessageExtraData`.